client/src/partials/admin/creditor_groups/creditor_groups.list.html

```diff
@@ -0,0 +1,32 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-8 col-md-offset-2">
+
+      <div ng-repeat="cg in CreditorGroupCtrl.creditorGroupList | orderBy : cg.name">
```

Comment:
It's always best to include a `track by` statement. [Documentation](https://docs.angularjs.org/api/ng/directive/ngRepeat#tracking-and-duplicates)